eta/core/datasets.py

```diff
@@ -1482,8 +1492,8 @@ def __init__(self, data_path, labels_path):
             data_path (str): path to data file
             labels_path (str): path to labels json
         '''
-        self._data_path = data_path
-        self._labels_path = labels_path
+        self.data_path = data_path
```

Comment:
The `__init__` docstring above reads "The label and data paths cannot and should not be modified after initialization." I think you should be creating and removing records, rather than modifying in place.

Reply:
I don't modify this in place.  The reason for making these variables public is so that they get serialized when the `BuilderDataset` is written to JSON.

Reply:
Fixed.